lib/resolver.js

```diff
@@ -0,0 +1,25 @@
+// Load modules.
+var NoOpenIDError = require('./errors/no_openid_error');
+
+
+/**
+ * Discover OpenID Connect provider configuration from Authentiq Provider.
+ *
+ * @return {Function}
+ * @api public
+ */
+function Resolver(options) {
+  options = options || {};
+
+  this.providerURL = options.providerURL;
+
+  if (!this.providerURL) {
+      throw new NoOpenIDError('No OpenID Connect issuer has been set');
```

Comment:
This will exit the Application at boot. We can move this to line 22 which will show an error message on each user sign-in. We think that is better for the developer to catch this early